manifests/moco-config.pp

```diff
@@ -310,9 +310,9 @@
             partner_min_version                   => 61,
             releaserunner_config_file             => 'release-runner.yml',
             allowed_branches                      => [ 'projects/jamun', 'projects/maple', 'projects/birch' ],
-            fennec_pattern                        => 'Fennec-.*',
-            firefox_pattern                       => 'Firefox-(5[89]|6[0-9]).*',
-            devedition_pattern                    => 'Devedition-(5[89]|6[0-9]).*',
+            # Exclude betas
+            fennec_pattern                        => 'Fennec-\d+\.\d+(\.\d+)?-.*',
+            firefox_pattern                       => 'Firefox-\d+\.\d+(\.\d+)?(esr)?-.*',
```

Comment:
I think we should drop the fennec/firefox changes in this hunk, to make using ship-it-v1 on maple easier.